src/test/scala/io/iohk/ethereum/validators/BlockHeaderValidatorSpec.scala

```diff
@@ -144,201 +147,201 @@ class BlockHeaderValidatorSpec extends FlatSpec with Matchers with PropertyCheck
     blockHeaderValidator.validate(blockHeaderWithInvalidNonce, validBlockParent) shouldBe Left(HeaderPoWError)
     blockHeaderValidator.validate(blockHeaderWithInvalidMixHash, validBlockParent) shouldBe Left(HeaderPoWError)
     blockHeaderValidator.validate(blockHeaderWithInvalidNonceAndMixHash, validBlockParent) shouldBe Left(HeaderPoWError)
-  }
+  }*/
 
-  it should "validate correctly a block whose parent is in storage" in new EphemBlockchainTestSetup {
-    blockchain.save(validBlockParent)
-    blockHeaderValidator.validate(validBlockHeader, blockchain) match {
-      case Right(_)  => succeed
-      case _ => fail
-    }
+it should "validate correctly a block whose parent is in storage" in new EphemBlockchainTestSetup {
```

Comment:
Previous indentation was correct - starting here, and continuing through the remainder of this file